manifests/init.pp

```diff
@@ -35,6 +35,9 @@
   $middleware_ssl_port       = '61614',
   $middleware_ssl            = false,
   $middleware_ssl_fallback   = false,
+  $middleware_ssl_cert       = undef,
+  $middleware_ssl_key        = undef,
+  $middleware_ssl_ca         = undef,
```

Comment:
This is a sensible name for this parameter, but the original parameter 'ssl_ca_cert' does become more confusing.  We're keeping it for backwards compatibility, but the ssl plugin doesn't need the client keys to be signed by any CA.  Often they are because most of the time people reuse puppet certs and keys, but as far as I know, plain ssl keys would work fine (I'll double check this later).